controllers/istio_controller.go

```diff
@@ -161,6 +161,8 @@ func (r *IstioReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		return ctrl.Result{}, err
 	}
 
+	values = mergeOverwrite(istio.Spec.GetRawValues(), values)
```

Comment:
I think this needs to go above `applyOverrides` so that the user can't change `global.istioNamespace` via `Istio.spec.rawValues`.